app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,215 @@
+require 'aws-sdk-s3'
```

Comment:
I think it should work without requiring.

Reply:
i test without and get error